lib/compilers/glimmer/index.js

```diff
@@ -42,3 +57,26 @@ export function getTemplateSpecifier(rootName, relativePath) {
 
   return `template:/${rootName}/${pathParts.join('/')}`;
 }
+
+function getManagerIdFromPath(path) {
+  let parts = path.split('.');
+  let managerId = null;
+
+  if (parts.length > 2) {
+    let lastIndex = parts.length - 1;
+    managerId = parts.splice(lastIndex - 1, 1)[0];
```

Comment:
Also, is having a `.` in a module name already allowed? I don't _think_ it is, but we should confirm